Design rules/Policy Design Rules.md

```diff
@@ -0,0 +1,42 @@
+# Policy Design Rules
+
+In dit document beschrijven we hoe we in het Kenniscentrum Standaarden van VNG Realisatie omgaan met Design Rules. 
+Daarbij doelen we op de eigen Design Rules, de Design Rules die in de Nederlandse API Strategie zijn beschreven maar ook op de Extensions die op beide niveaus beschikbaar zijn.
+Extensions hebben geen officiële status, ze kunnen worden gezien als potentiële Design Rules.
+
+## Toepassing Design Rules
+
+Het toepassen van Design Rules en Extensions is aan de volgende regels gebonden:
+
+1. Bij het vervaardigen van API specificaties houden we ons aan [de Design Rules zoals gedefinieerd in de Nederlandse API Strategie](https://docs.geostandaarden.nl/api/API-Designrules/);
+2. Daarbij kijken we tevens in hoeverre [de Extensions van de Nederlandse API Strategie](https://docs.geostandaarden.nl/api/API-Strategie-ext/) van toepassing zijn. In een Design Decision van het project waarin de API specificatie wordt vervaardigd beschrijven we aan welke Extensions we ons houden;
+3. Tevens houden we ons aan [de Design Rules van VNG Realisatie](https://github.com/VNG-Realisatie/API-Kennisbank/blob/master/Design%20rules/readme.md);
```

Comment:
Ik zie dit anders Robert, in het kort geformuleerd: 
1. We volgen de design rules van de landelijke API-strategie.
2.  We volgen de VNG Design Rules.
3.  Waar mogelijk volgen we de landelijke Design Rule extensions. Als we een extension gebruiken waar nog geen VNG Design Rule voor bestaat, dan wordt deze opgenomen in de VNG Design Rules, eventueel aangescherpt vanuit onze eigen visie.

In mijn beleving hebben we geen VNG Extensions. Wij zijn met 6 man en kunnen uitstekend overeenstemming bereiken over een Design Rule en hoeven dat niet op de lange baan te schuiven. 
Voor het bovenstaande geldt : Als je vanuit project-belang overweegt om een Design Rule (landelijk of VNG) niet toe te passen dan documenteer je je argumenten waarom dat binnen het project niet gedaan wordt en die argumenten worden binnen het team besproken en afgezegend.

Reply:
Eens, behalve met regel 3. Ik vind dat je binnen een project ook een landelijke extension moet kunnen gebruiken zonder dat we deze opnemen in onze eigen Design Rules.